### ISSUE TITLE
UID объекта в запросе

### ISSUE THREAD

**Issue by Eremkinig** [SeiOkami/OneS#5]
В интеграционных задачах, когда требуется отдать в качестве ответа на GET-запрос, например, uid номенклатуры, приходится в цикле перебирать и получать его. А это уже дополнительное время.

**Comment by VolAll** [comment]
+1  это прям праздник был бы.

**Comment by ellavs** [comment]
Да, очень нужно, а то прямо "боль" при написании сервисов интеграции с внешними системами :)

**Comment by SeiOkami** [comment]
Добавил про возможность приведения УИД к строке

**Comment by SeiOkami** [comment]
Отправлено боту 1С 06.04.2021 в 12:36

**Comment by Eremkinig** [comment]
Наши и не только наши просьбы услышали и в 21-ой версии анонсировали. Только вот еще год-два придется подождать стабильно работающей версии данного релиза

**Comment by marmyshev** [comment]
Одно из самых популярных пожеланий. 

Вот, говорят, сделают https://wonderland.v8.1c.ru/blog/poluchenie-unikalnogo-identifikatora-obekta-iz-ssylki-v-zaprose/

**Comment by SeiOkami** [comment]
Появлось в 8.3.22:
- [Разбор на ютуб про баги](https://youtu.be/1rywI70iScc) 
- [Разбор недостатков на Инфостарт ](https://infostart.ru/public/1788844/?ref=1159)
